larky/src/main/java/com/verygood/security/larky/modules/SysModule.java

```diff
@@ -0,0 +1,29 @@
+package com.verygood.security.larky.modules;
+
+import java.nio.ByteOrder;
+
+import net.starlark.java.annot.StarlarkBuiltin;
+import net.starlark.java.annot.StarlarkMethod;
+import net.starlark.java.eval.StarlarkValue;
+
+
+@StarlarkBuiltin(
```

Comment:
Please add description for this class. It's purpose is unclear

Reply:
It was in the commit message, but I will put it in the header. Was going to update it with this:

```A very simple module that just exposes basic information for portability
reasons.

This module will just expose things like `maxint` or `byteorder`, just to make
it easier to port certain things.```

Any objections?

Reply:
Addressed this here: https://github.com/verygoodsecurity/starlarky/pull/150/commits/fcfa911ec73e04e348051a6b8a1edb06d1c629d4. 

Plz merge https://github.com/verygoodsecurity/starlarky/pull/150 first and this should resolve this conversation.